src/index.ts

```diff
@@ -3,18 +3,20 @@ import { CommonModule } from '@angular/common';
 
 import { KeyboardShortcutService } from './keyboard-shortcut.service';
 import { KeyboardShortcutDirective } from './keyboard-shortcut.directive';
+import { IKeyboardShortcutConfig } from 'libraries/keyboard-shortcut-configuration.library';
```

Comment:
This should be added as a relative import './libraries/keyboard-shortcut-configuration.library' otherwise there can be some issues with imports during usage.